### PR TITLE
Code refactoring in WindowsSpecification

### PR DESCRIPTION
**Pull request by EOKruglov**
Linked issues: 
* https://github.com/pywinauto/pywinauto/issues/710
* https://github.com/pywinauto/pywinauto/issues/807

Changes:
* rename .wrapper_object() to .find(); wrapper_object marked as deprecated; WrapperObject was removed.
* rename .child_window() to .by(); child_window, window marked as deprecated ChildWindow, Window, Window_, window_ were removed
* Exists, Wait, WaitNot, PrintControlIdentifiers were removed

### CODE REVIEW

File: apps/Gtk_samples/gtk_example.py [hunk 1]
```diff
@@ -100,7 +100,7 @@ def _create_textview(self):
         return scrolledwindow
 
     def __init__(self):
-        Gtk.Window.__init__(self, title="MainWindow")
+        Gtk.by.__init__(self, title="MainWindow")
```
Comment (vasily-v-ryabov) [inline apps/Gtk_samples/gtk_example.py:103]: Looks like wrong replace. This is not connected with the core library code.

File: sandbox/recorder.py [hunk 2]
```diff
@@ -84,8 +84,8 @@ def OnKeyboardEvent(event):
 
     app = Application()
 
-    if event.Window:
-        wrapped = app.window(handle = event.Window)
+    if event.by:
+        wrapped = app.window(handle = event.by)
```
Comment (vasily-v-ryabov) [inline sandbox/recorder.py:88]: This file shouldn't be touched as well.

**Review by vasily-v-ryabov — CHANGES_REQUESTED** [review]

**Review by vasily-v-ryabov — APPROVED** [review]

**Comment by vasily-v-ryabov** [comment]
`import warnings` is not used any more in `base_application.py`. So it can be removed.

**Comment by EOKruglov** [comment]
> `import warnings` is not used any more in `base_application.py`. So it can be removed.

removed

**Pull request merged**